### PR TITLE
Ready for code review

### PR DESCRIPTION
**Pull request by noomdalv**
Please review, thanks.

### CODE REVIEW

**Comment by mosaaleb** [comment]
Currently reviewing

File: index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+          <li id="macTab"><a href="#"><span></span></a></li>
+          <li id="ipodTab"><a href="#"><span></span></a></li>
+          <li id="iphoneTab"><a href="#"><span></span></a></li>
+          <li id="ipadTab"><a href="#"><span></span></a></li>
+          <li id="iTunesTab"><a href="#"><span></span></a></li>
+          <li id="supportTab"><a href="#"><span></span></a></li>
+        </ul>
+
+        <div id="navSearch">
+          <div class="searchBar">
```
Comment (mosaaleb) [inline index.html:27]: You can use an `<input>` tag without wrapping it with `<form>` tag for simple UI componenet like search bar. See [this](https://stackoverflow.com/questions/6483622/text-field-without-form?rq=1) for more information

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @noomdalv :wave:

### Amazing work so far :clap:. You have handled most of the requirements, yet there still some requirements needs be met.



### Main content links
- [ ] Each link is an `<a>` HTML tag, with a `<div>` tag inside it.
- [ ] The `<div>` tag  inside the `<a>` tag should use an image as background --  The text on the original page is actually an image

### Bottom images 
- [ ] is made of four `<a>` tags with `<div>` tags inside it.
- [ ] The `<div>`s inside the `<a>` tags should use an image as a background

### General
- [ ] Let's make sure that our style guidelines are ok by using [stickler](https://github.com/microverseinc/linters-config/tree/master/css). It's very easy to use, just follow the link.

**Also see my comment below**

Please submit a new code review once you finish these requirements.

#### Happy coding :computer:

**Comment by noomdalv** [comment]
hey man thanks for the review,

gotta admit im confused because i thought the idea was to make the page as
similar as the original and the structure of my project is just as theirs
o_O

so here i am thinking if theres a list of these specific requirements for
each project? the odin project page just list some basic ones but thats
it.. not much regarding html structure :/
i was reading about stickler just now, because the only app to check for
errors on my last 3 projects has been w3c validator, maybe stickler is
listing these requirements?
im trying to install stickler and run it on my repo but i get
"We cannot generate a configuration file for the languages in this
repository. Please create a .stickler.yml".
am i doing it wrong? please help

**Comment by mosaaleb** [comment]
Hi @noomdalv 
We have in Microverse a list for requirements written carefully for every project so that all the projects will have better html & css structure. Applying best practices is very important step in your learning journey.

Regarding stickler, we are using `style lint` in `.sticker.yml` file so that you have good style guidelines when writing css files. It does not though check for html files, but you can install html lint extension available in all text editors instead of using external validator. If you are using *vscode*, there is an extension called [htmllint](https://marketplace.visualstudio.com/items?itemName=mkaufman.HTMLHint), it will save you time when validating html files.

For the error part, have you created the `.stickler.yml` file in your project home directory?
All the instructions are explained in here [stickler](https://github.com/microverseinc/linters-config/tree/master/css)

Contact me on slack (Muhammad Ebeid) if anything is still unclear.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+      <section id="bottomImages">
+
+        <a href="#"><div class="btmImg bi1"></div></a>
+
+        <a href="#"><div class="btmImg bi2"></div></a>
+
+        <a href="#"><div class="btmImg bi3"></div></a>
+
+        <a href="#"><div class="btmImg bi4"></div></a>
```
Comment (Oghenebrume50) [inline index.html:68]: Hi great work done so far, while everything is setup properly, this container overlaps to the bottom you have to make a little bit responsive. Do that and submit for re-review, kudos for a great start. And please check out Github flow, you should be using a feature brance not development branch. Anyways you can leave Github flow until your next project

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (noomdalv) [reply index.html:68]: done, imgs should have width percentages to resize accordingly

**Comment by samgaco** [comment]
### Hi @noomdalv :hand: 

### Great job so far :+1: The website looks really nice. Still some points to address before giving you the approval:

* Correct the errors in the index file given to you by the https://validator.w3.org
* Enable a [focus effect](https://www.w3schools.com/cssref/sel_focus.asp) for the input tag of your search bar.
* For the text **_" Watch the "Your Verse" film and explore the stories"_** , that should be in text format and it should have a hover underline effect.
* Add a project description to the **readme file**, it is also suggested to add your contact and a live demo of the site. [Here](https://github.com/othneildrew/Best-README-Template) there is an example template for inspiration purposes.

### That would be it. Please :pray: address these changes and submit back for another code review.

**Comment by noomdalv** [comment]
Hi, i went ahead an added the focus effect for the search input and the
main readme file was on master branch, so i just added a new one updated to
the pull request from development, there arent any wc3validator errors
showing for me, just warnings about headings.
i have an issue with this point:

   - For the text *" Watch the "Your Verse" film and explore the stories"* ,
   that should be in text format and it should have a hover underline effect.

Why? Because these are the project fixes/requirements that i have on my
first review from TSE *@mosaaleb*
Main content links

   -  There should be three links
   -  Each link is an <a> HTML tag, with a <div> tag inside it.
   -  The <div> tag inside the <a> tag should use an image as background --
   The text on the original page is actually an image

i sent him a msg regarding these suggestions and he said its microverse
specific requirements for the project, but now you tell me i need to put
text and not an image?
i dont understand why you guys are contradicting yourselves, please let me
know what should i do next so my project can be completed.

**Comment by Oghenebrume50** [comment]
Hi @noomdalv, I guess you must have seen the comment from @samgaco the intention was not to confuse you, we apologize for any frustration. As it stands now you have met all the requirements and you can go ahead to submit this project. Good luck with your next project. I would like to point out that you read up on [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) and use it for all other HTML projects.

**Comment by samgaco** [comment]
@noomdalv  @Oghenebrume50  

Since I am quoted, I'd like to clarify my requirement; in the original website the first two texts are images, so @noomdalv  did well on that, but as it shows in the original website, the third one: **Watch the “Your Verse” film and explore the stories**, it's a text and it has a hoover underline effect. My intention was not to contradict Raphael's review, but I apologize to both anyway if it seemed like it.

I am sorry for the misunderstanding. Regards.

**Comment by noomdalv** [comment]
Hello, i would like to object to this comment:
"the third one: *Watch the “Your Verse” film and explore the stories*, it's
a text and it has a hoover underline effect."

hmm not really, according to the original website
https://web.archive.org/web/20140301004610/http://www.apple.com/ every
"text" line you see on the main content its actually an ,
this specific image contains 2 versions of "*Watch the “Your Verse” film
and explore the stories*" , the first one has no underline and the second
has the underline, here's a snippet of said image:

[image: image.png]

and heres a snippet of the original html structure:

[image: image.png]


The way they achieve the underline effect its simply by positioning the
image with negative top margin when its hovered.

Its not my intention to disagree with your comments but to clarify
everything because my original project html structure was based on the same
as the original website but the first TSE told me i had to make changes
that werent explicitly listed, i complied with his suggestions and 2
reviews later im being told to change it again, so its really frustrating
to me because im just basically wasting time and losing motivation 😐.

Tbh now im just worried because i dont know if for the next code review i
will get another TSE and he will tell mesomething different, please
consider my position, thanks.

I added the underline effect as the original website and it should work
now, should i submit another code review form?

**Comment by samgaco** [comment]
Hi @noomdalv 

@Oghenebrume50 approved your project already in his last message. You are right, I apologize for my mistake. I am very sorry about it. Regards :+1:

**Pull request merged**